tools/cldr-apps/src/main/webapp/js/new/cldrSurvey.js

```diff
@@ -1366,22 +1366,6 @@ const cldrSurvey = (function () {
     surveyUserCov = cov;
   }
 
-  function getSurveyOrgCov() {
-    return surveyOrgCov;
```

Comment:
duplication again, discovered by webpack again

Reply:
very nice.  

basically, the module system is enforcing some discipline in the namespace.